test/test_coriolis.jl

```diff
@@ -12,15 +12,16 @@ function instantiate_fplane_2(FT)
 end
 
 function instantiate_ntfplane_1(FT)
-    coriolis = NonTraditionalFPlane(FT, fz=π, fy=π)
-    @test coriolis.fz == FT(π)
-    @test coriolis.fy == FT(π)
+    coriolis = GeneralFPlane(FT, coriolis_frequency=1, rotation_axis=[0, cosd(45), sind(45)])
```

Comment:
After you do this for the `FPlane` are you thinking of doing the `BetaPlane` too?  I'm guessing it would be easier than this and might be nice to have.

Reply:
I agree it would be easier to have, but I don't intend on doing this in this PR. I've also never run a simulation with the Beta plane and only study small scale turbulence (where the beta effect doesn't matter) so it might be best to leave that endeavor to someone who has more familiarity with it...

Reply:
I did not mean to suggest you do it here.  That would be better saved for a later PR, and should be revistied afterwards.  But it would make sense to be consistent across all of our planes, long term.